frontend/src/browser/users/EditUser.js

```diff
@@ -626,27 +674,26 @@ export class EditUser extends Component {
                     />
 
                     {mode !== 'profile' ?
-                      <Field
-                        name="userState"
-                        component={this.renderSelect}
-                        label={`${formatMessage(messages.userState)}`}
-                      >
-                        <option value={'active'}>
-                          {formatMessage(messages.activeUserState)}
-                        </option>
-                        <option value={'inactive'}>
-                          {formatMessage(messages.inactiveUserState)}
-                        </option>
-                        <option value={'temporarySuspended'}>
-                          {formatMessage(messages.temporarySuspendedUserState)}
-                        </option>
-                        <option value={'temporarySuspended'}>
-                          {formatMessage(messages.expelledUserState)}
-                        </option>
-                        <option value={'temporarySuspended'}>
-                          {formatMessage(messages.endedUserState)}
-                        </option>
-                      </Field>
+                      <div>
+                        <Field
+                          name="state"
+                          component={this.renderSelect}
+                          label={`${formatMessage(messages.userState)}`}
+                        >
+                          {userStates ? Object.keys(userStates).map((key) =>
+                            <option value={userStates[key]}>
+                              {formatMessage(messages[`${userStates[key]}UserState`])}
+                            </option>
+                          ) : ''}
+                        </Field>
+
+                        <Field
+                          name="monthlySchoolFee"
```

Comment:
lenže ako to rozumne naprogramovať, sa mi to vôbec nepáči, že sa to všade vypisuje x /100 a pri zadávaní to bude ešte menej intuitívne. Možno by to chcelo mať na frontende v eurách a prevádzať to na serveri